app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -55,7 +55,7 @@ class Meta:
         model = NimbusExperiment
 
     @classmethod
-    def create_with_status(cls, target_status, **kwargs):
+    def create_with_status(cls, target_status, **kwargs) -> NimbusExperiment:
```

Comment:
Adding type annotations?

Reply:
Habit, I can leave it off, but its nice having VS Code use proper command completion which sorta works when these are added.

Reply:
Yeah pylance will pick them up. I've been trying to figure out how to add type annotations in a nice way and I've been wondering whether we can get away with adding type annotations for just the nimbus stuff, but there's enough overlap with legacy stuff I don't think we can do it for just Nimbus and actually enforce it.  I'd prefer if we do it to go all in on it instead of making it haphazard, but that's just my preference.